modules/orchestrator/endpoints/microservice.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // ListMicroServiceProject 获取使用微服务的项目列表
 func (e *Endpoints) ListMicroServiceProject(ctx context.Context, r *http.Request, vars map[string]string) (httpserver.Responser, error) {
-	_, err := user.GetUserID(r)
+	userId, err := user.GetUserID(r)
```

Comment:
There is no need to modify the microservice project api provided by orchestrator. It should be the msp module that calls this api, and then aggregates tenant data and old data